code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -196,6 +228,14 @@
 /obj/machinery/computer/pandemic/proc/create_culture_bottle(index)
 	var/id = get_virus_id_by_index(text2num(index))
 	var/datum/disease/advance/adv_disease = SSdisease.archive_diseases[id]
+
+	var/list/cures = get_beaker_cures(id)
+	if(!cures.len)
```

Comment:
This should not be early return. Even if we don't have cures the culture bottle should still be printed